WikiCohereEnglishEmbeddingOnlyStreamingScenario.cs

```diff
@@ -1,17 +1,20 @@
-﻿using Microsoft.Extensions.Configuration;
+﻿using Microsoft.Azure.Cosmos;
+using Microsoft.Extensions.Configuration;
 
 namespace VectorIndexScenarioSuite
 {
-    internal class WikiCohereEnglishEmbeddingOnlyStreamingScenario : WikiCohereEmbeddingOnlyBaseSceario
+    internal class WikiCohereEnglishEmbeddingOnlyStreamingScenario : WikiCohereEnglishEmbeddingOnlyScenario

```

Comment:
Hey lets not have Streaming inherit from embedding only scenario. They are different. For instance both have different 'ComputeInitialAndFinalThroughput' requirments.

Instead, WikiCohereEnglishEmbeddingOnlyStreamingScenario and WikiCohereEnglishEmbeddingOnlyScenario should inherit from a WikiCohereEnglishEmbeddingBaseScenario with common declarations.